external-providers/java-external-provider/pkg/java_external_provider/service_client.go

```diff
@@ -57,7 +57,7 @@ func (p *javaServiceClient) Evaluate(ctx context.Context, cap string, conditionI
 		return provider.ProviderEvaluateResponse{}, fmt.Errorf("unable to get query info: %v", err)
 	}
 	// filepaths get rendered as a string and must be converted
-	if len(cond.Referenced.Filepaths) > 0 {
+	if cond.Referenced.Filepaths != nil && len(cond.Referenced.Filepaths) > 0 {
```

Comment:
doesn't len() do an implicit nil check?

Reply:
I thought so - but I was hitting a nil panic error on this line in kantra. After adding this in, it fixed that.